src/Data/Parameterized/Context.hs

```diff
@@ -185,6 +187,26 @@ dropPrefix xs0 prefix err = go xs0 (sizeInt (size xs0))
       Just Refl -> success Empty
       Nothing   -> err
 
+-- | Flattens a nested assignment over a context of contexts @ctxs :: Ctx (Ctx
+-- a)@ into a flat assignment over the flattened context @CtxFlatten ctxs@.
+flattenAssignment ::
+  Assignment (Assignment f) ctxs ->
+  Assignment f (CtxFlatten ctxs)
+flattenAssignment ctxs =
+  case viewAssign ctxs of
+    AssignEmpty -> empty
+    AssignExtend ctxs' ctx -> flattenAssignment ctxs' <++> ctx
+
+-- | Given the size of each context in @ctxs@, returns the size of @CtxFlatten
+-- ctxs@.  You can obtain the former from any nested assignment @Assignment
+-- (Assignment f) ctxs@, by calling @fmapFC size@.
+flattenSize ::
```

Comment:
This isn't building because of the `Ctx` import prefix.  Looks like you copied this from somewhere else and didn't clean it up.

Reply:
Indeed, fixed in 6dc5c48b3aff5c1dd86035d445997dea3d4c7646